packages/matchbox/src/components/ProgressBar/ProgressBar.js

```diff
@@ -1,45 +1,51 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
-import styles from './ProgressBar.module.scss';
-
-class ProgressBar extends Component {
-  static displayName = 'ProgressBar';
-
-  static propTypes = {
-    /**
-     * Completion in percentage
-     */
-    completed: PropTypes.number.isRequired,
-
-    /**
-     * Bar color
-     */
-    color: PropTypes.oneOf(['orange', 'blue', 'navy', 'purple', 'red'])
-  };
-
-  static defaultProps = {
```

Comment:
Just so we don't forget later, could we make a note of the default prop changes in `unreleased.md`?